src/main/java/ch/njol/skript/conditions/CondIsUnbreakable.java

```diff
@@ -19,38 +19,47 @@
  */
 package ch.njol.skript.conditions;
 
-import org.bukkit.inventory.meta.ItemMeta;
-
-import ch.njol.skript.Skript;
 import ch.njol.skript.aliases.ItemType;
 import ch.njol.skript.conditions.base.PropertyCondition;
 import ch.njol.skript.doc.Description;
 import ch.njol.skript.doc.Examples;
 import ch.njol.skript.doc.Name;
-import ch.njol.skript.doc.RequiredPlugins;
 import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
 
 @Name("Is Unbreakable")
 @Description("Checks whether an item is unbreakable.")
-@Examples("if event-item is unbreakable")
-@Since("2.5.1")
-@RequiredPlugins("Minecraft 1.11+")
+@Examples({
+	"if event-item is unbreakable:",
+		"\tsend \"This item is unbreakable!\" to player",
+	"if tool of {_p} is breakable:",
+		"\tsend \"Your tool is breakable!\" to {_p}"
+})
+@Since("2.5.1, INSERT VERSION (breakable)")
 public class CondIsUnbreakable extends PropertyCondition<ItemType> {
 	
 	static {
-		if (Skript.methodExists(ItemMeta.class, "isUnbreakable")) {
-			register(CondIsUnbreakable.class, "unbreakable", "itemtypes");
-		}
+		register(CondIsUnbreakable.class, "[:un]breakable", "itemtypes");
 	}
-	
+
+	private boolean negated;
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		negated = !parseResult.hasTag("un");
```

Comment:
It would probably be better to call `setNegated` here. Note that you would need to account for it already being negated (e.g. `<item> is not breakable`)

Reply:
I don't necessarily think so because the condition isn't actually negated there it's just testing for a different thing.